### ISSUE TITLE
"Symbol void" error on first export (doom)

### ISSUE THREAD

**Issue by nazzacode** [kjambunathan/org-mode-ox-odt#239]
I would like to use this package (doom emacs). 

I have set the package archive and installed as by the instructions. 
As it wasn't working, I added the doom specific override.

Now it has loaded but I get the following error: 

```
condition-case: OpenDocument export failed with error: ‘Symbol’s function definition is void: dom-remove-node’
```

Org version: 9.6

I'm no elisp whizz, what is going on?

**Comment by kjambunathan** [comment]
Looks like

(require 'dom)

is missing


Do

M-x load-library dom

and export again it should work


Dom library comes with Emacs, and is used by eww.


-----

Since you are using my ELPA archive, you should have already downloaded
odt.el, and loading dom should be sufficient.


If you are using straight, ensure that you add odt.el to the straight
recipe.  See bug report
https://github.com/kjambunathan/org-mode-ox-odt/issues/227.

**Comment by nazzacode** [comment]
That worked. Thank you most kindly :)

**Comment by nazzacode** [comment]
Any idea how I can get this function to be permanent, rather than reverting in a new instance?

**Comment by kjambunathan** [comment]
I have released `ox-odt-9.6.620` with a bugfix for this issue.  You can either do an upgrade via the package manager, or if you don't want to upgrade make the oneline change that you see here to `odt.el' https://github.com/kjambunathan/org-mode-ox-odt/commit/dfb12db9a88fc8bedb7d8f6c0b099be1f1e3838d

Thanks for reporting this issue.  I am not sure why `org'-s `make' wasn't catching the missing dependency in the first place.

**Comment by kjambunathan** [comment]
If your are changing `odt.el' locally,   remember to do `M-x byte-compile-file' ...
